esterni/manuale-sviluppatore/components/descrizione-generale.tex

```diff
@@ -0,0 +1,41 @@
+\documentclass[../manuale-utente.tex]{subfiles}
```

Comment:
Questa descrizione generale è la stessa che c'è in manuale-utente (corretta).
Cercate di segnalare eventuali errori solo in questa DR, così da non sparpagliare i commenti.